src/client/sky.cpp

```diff
@@ -206,16 +209,17 @@ void Sky::render()
 
 		// Draw far cloudy fog thing blended with skycolor
 		for (u32 j = 0; j < 4; j++) {
-			video::SColor c = cloudyfogcolor.getInterpolated(m_skycolor, 0.45);
+			video::SColor c =
+					cloudyfogcolor.getInterpolated(m_skycolor, 0.45);
```

Comment:
Please only wrap lines at 90 columns (tab size 4). This may be a bad clang format suggestion.